run-cron.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 # Example cron script. The actual cron script on wpackagist.org is more complex, and involves ensuring generated files have the correct permissions.
-php bin/cmd refresh
-php bin/cmd update
-php bin/cmd build
+php bin/console migrate
```

Comment:
If this turns out to be hard to change live we could use/alias `cmd` but I think following the normal Symfony convention will be less confusing long term.